templates/taxi/manufacturer_confirm_delete.html

```diff
@@ -7,4 +7,4 @@ <h1>Delete manufacturer?</h1>
 
     <input type="submit" value="Yes" class="btn btn-danger">
```

Comment:
not obligatory but it would be more comfortable for user if this form had button "Cancel" or "No"